code/08-lifecycle-hooks.js

```diff
@@ -1 +1,64 @@
 // 08-lifecycle-hooks
+
+/**
+ * Get the reactivation options for inactive policies.
+ * @param {PlatformPolicy} policy The policy to be reactivated.
+ * @return {ReactivationOption[]} One of these options must be selected whenever an inactive policy is reactivated.
+ */
+const getReactivationOptions = (policy) => {
+  const settlementAmount = policy.balance < 0 ? -policy.balance : 0;
+  return [
+    new ReactivationOption({
+      type: 'reinstatement',
+      settlementAmount,
+      description:
+        'For a policy to be reinstated, all arrear premiums will immediately be due.',
+      minimumBalanceRequired: true,
+    }),
+    new ReactivationOption({
+      type: 'recommencement',
+      description:
+        'For a policy to be recommenced, all arrear premiums will be deducted from the first claim income.',
+      minimumBalanceRequired: false,
+    }),
```

Comment:
Not necessary to have both, one option should suffice here! 😄 

We sometimes use two hooks if they have different logic associated with them, so you're able to write separate logic using lifecycle hooks.